pkg/provision/converter.go

```diff
@@ -8,17 +8,24 @@ import (
 
 	"github.com/threefoldtech/zos/pkg"
 	"github.com/threefoldtech/zos/pkg/container/logger"
-	generated "github.com/threefoldtech/zos/pkg/gedis/types/provision"
 	"github.com/threefoldtech/zos/pkg/network/types"
+	"github.com/threefoldtech/zos/tools/bcdb_mock/models/generated/workloads"
 )
 
 // ContainerToProvisionType converts TfgridReservationContainer1 to Container
-func ContainerToProvisionType(c generated.TfgridReservationContainer1) (Container, string, error) {
+func ContainerToProvisionType(c workloads.TfgridWorkloadsReservationContainer1) (Container, string, error) {
+	env := func(m map[string]interface{}) map[string]string {
+		o := make(map[string]string)
```

Comment:
```suggestion
		o := make(map[string]string,len(m))
```
So we avoid to allocated potentially multiple time during the loop